### ISSUE TITLE
Add featured categories

### ISSUE THREAD

**Issue by dragly** [ComputationalPhysics/atomify-lammps#108]

**Comment by andeplane** [comment]
I assume we can just (for now) add
name: "Featured",
simulations: [
    "qrc:/simulations/water/singlewater",
    "qrc:/simulations/silica/zeolite_zsm5",
    "qrc:/simulations/diffusion/simple_diffusion"
]
etc?

**Comment by dragly** [comment]
Yes. That will do. No reason to make a special case for this at the moment (or maybe not even later).
